corehq/apps/app_manager/tests/views/test_cli.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+from django.test import SimpleTestCase
+from unittest.mock import patch, MagicMock
+from django.test.client import RequestFactory
+
+from ...views.cli import direct_ccz
+from ...views import cli
+
+
+class DirectCCZTests(SimpleTestCase):
+    def test_missing_app_id(self):
+        rf = RequestFactory()
+        request = rf.get('', {})
+        response = direct_ccz(request, 'test-domain')
+
+        self.assertEqual(response.status_code, 400)
+        response_reason = json.loads(response.content)
+        self.assertEqual(response_reason['message'], 'You must specify `app_id` in your GET parameters')
+
+    @patch.object(cli, '_get_app', return_value=None)
```

Comment:
I've never seen `patch.object` used on a python module before. Is there a benefit of this approach over  `patch("corehq.apps.app_manager.views.cli._get_app", return_value=None)`

Reply:
I've generally been using `patch.object` for everything, just because I feel like strings are much more brittle. This way, I get intellisense in my IDE, and if code was shifted around, the editor would show that and possibly be able to automatically correct it. What is the upside of the other approach?

Reply:
I think the only (potential) advantage of the other approach is that you don't need to import the module which is moot in this case.